pfsimulator/clm/clm.F90

```diff
@@ -315,7 +315,7 @@ subroutine clm_lsm(pressure,saturation,evap_trans,topo,porosity,pf_dz_mult,istep
      !=== Initialize CLM and DIAG variables
      if (clm_write_logs==1) write(999,*) "Initialize CLM and DIAG variables"
      do t=1,drv%nch 
-        clm%kpatch = t
+        clm(t)%kpatch = t

```

Comment:
Commits should be targeted at a single issues.   Either CLM patch needs to be removed from the commit or this won't be merged until the CLM PR has been approved.

Reply:
CLM Patch was aproved.  This was fixed in #272